config/examples/BigTreeTech/SKR Mini E3 1.2/Configuration_adv.h

```diff
@@ -1880,7 +1880,7 @@
   #define INTERPOLATE       true  // Interpolate X/Y/Z_MICROSTEPS to 256
 
   #if AXIS_IS_TMC(X)
-    #define X_CURRENT       580        // (mA) RMS current. Multiply by 1.414 for peak current.
+    #define X_CURRENT       600        // (mA) RMS current. Multiply by 1.414 for peak current.
```

Comment:
Let's keep this consistent with other PRs/Creality configs. See https://github.com/MarlinFirmware/Marlin/pull/15115 & https://github.com/MarlinFirmware/Marlin/pull/15666.

Reply:
is reverted in current commit